app/service/rest.py

```diff
@@ -154,19 +153,24 @@ def get_service_by_id(service_id):
 @service_blueprint.route('', methods=['POST'])
 def create_service():
     data = request.get_json()
-    if not data.get('user_id', None):
-        errors = {'user_id': ['Missing data for required field.']}
+    errors = {
+        required_field: ['Missing data for required field.']
+        for required_field in ['user_id', 'free_sms_fragment_limit']
+        if not data.get(required_field, None)
+    }
```

Comment:
data.get(required_field) without 'None' in the second argument seems to achieve the same thing.